tasks/src/text-to-image/about.md

```diff
@@ -25,6 +25,9 @@ You can contribute variants of this task [here](https://github.com/huggingface/h
 You can add a small snippet [here](https://github.com/huggingface/hub-docs/blob/main/tasks/src/text-to-image/about.md) that shows how to infer with `text-to-image` models.
   
 ## Useful Resources
+- [Hugging Face Diffusion Models Class](https://github.com/huggingface/diffusion-models-class)
```

Comment:
After discussing with @osanseviero we think it's good to use a standard "course" term for all our education materials
```suggestion
- [Hugging Face Diffusion Models Course](https://github.com/huggingface/diffusion-models-class)
```